README.md

```diff
@@ -47,3 +45,25 @@ Feel free to send some Monero to the address below. And if you do so, feel free
 XMR: `8ADbBKaunVWjdg5aWQ5ZBNDACdPVMTUBnKETaZbUZ8gMfDfpwhcBeo31kfUgCJKATMPaqmsUoxBwicTpRLg4p4F57kPJ5ab`
 
 For more, see our [Contributions document](CONTRIBUTING.md)
+
+## Building
```

Comment:
Build instructions - Niiiice!

Reply:
Honestly, I want to include a bit of a "beginner's guide" at some point, linking to good resources to learn more about various things Fuzzr does and uses, links to documentation for a very beginner to learn how to check out git repos, and basic platform-specific instructions on how to build Fuzzr for their respective platforms. Some basic troubleshooting could happen, also.

Reply:
And of course, eventually system-specific binaries will make thinks even easier